nexus/external-api/src/lib.rs

```diff
@@ -1313,7 +1313,7 @@ pub trait NexusExternalApi {
         path_params: Path<params::AffinityInstanceGroupMemberPath>,
     ) -> Result<HttpResponseOk<AffinityGroupMember>, HttpError>;
 
-    /// Add a member to an affinity group
+    /// Add member to affinity group
```

Comment:
In theory these could go `Add affinity group member`, but I think in this case the `to` is worth it. "add member" is a nice clean phrase to have up front.